sdks/python/apache_beam/examples/inference/sklearn_mnist_classification.py

```diff
@@ -104,8 +104,7 @@ def run(
 
   label_pixel_tuple = (
       pipeline
-      | "ReadFromInput" >> beam.io.ReadFromText(
-          known_args.input, skip_header_lines=1)
```

Comment:
skip_header_lines is used, just in case if the csv file has column names.

Reply:
I think its a pretty surprising experience, especially since we don't have headers in our example csv. I called out that the created csv shouldn't have headers though.